### DISCUSSION TITLE
The Faceted search module doesn't allow me to double filter

### DISCUSSION THREAD

**Discussion by webtimeorologi** [PrestaShop/PrestaShop#36148] [Q&A]
We have this problem on our website www.webtimeorologi.it. If we try to select 2 filters at the same time, the module deletes the first one. For example, if we go to JEWELRY - MORELLATO and select BRACELET and then want to add MAN, the module eliminates the first selection BRACELET

 Prestashop Versione 8.0.4 Versione PHP: 7.4.33   <img width="1421" alt="Screenshot 2024-05-14 alle 12 16 30" src="https://github.com/PrestaShop/PrestaShop/assets/169664501/881e5bc3-755b-4069-8abb-ad893ec8b2cb"> <img width="1482" alt="Screenshot 2024-05-14 alle 12 16 43" src="https://github.com/PrestaShop/PrestaShop/assets/169664501/0294bdde-7ef6-4097-9f10-00fabdd23c38">

**Comment by webtimeorologi** [comment]
Please help me :(

**Comment by SharakPL** [comment]
@webtimeorologi looks like your theme uses custom filters module instead of native **ps_facetedsearch**. Please check your module's configuration page. There should be an option to allow multiple filters. If there isn't one then you should contact the author of the module/theme on how to fix this.

**Comment by Hlavtox** [comment]
This is a theme issue, contact the author.
